src/modules/Builder/features/BuilderAppletSettings/IntegrationsListSetting/Integrations/LorisIntegration/ConfigurationPopup/ConfigurationPopup.tsx

```diff
@@ -0,0 +1,187 @@
+import { useCallback, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Control, FieldValues, FormProvider, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Box } from '@mui/material';
+
+import { applet, banners } from 'shared/state';
+import { Modal } from 'shared/components/Modal';
+import { Svg } from 'shared/components';
+import {
+  StyledBodyMedium,
+  StyledModalWrapper,
+  StyledTitleMedium,
+  theme,
+  variables,
+} from 'shared/styles';
+import { useAppDispatch } from 'redux/store';
+import { IntegrationTypes } from 'shared/consts';
+
+import {
+  ConfigurationForm,
+  ConfigurationPopupProps,
+  ConfigurationsSteps,
+} from './ConfigurationPopup.types';
+import { configurationFormSchema } from './ConfigurationPopup.schema';
+import { getScreens } from './ConfigurationPopup.utils';
+import { fetchLorisProjects, saveLorisProject } from '../LorisIntegration.utils';
+
+export const ConfigurationPopup = ({ open, onClose }: ConfigurationPopupProps) => {
+  const { t } = useTranslation();
+  const methods = useForm<ConfigurationForm>({
+    resolver: yupResolver(configurationFormSchema()),
+    defaultValues: {
+      hostname: '',
+      username: '',
+      password: '',
+      project: '',
+    },
+  });
+
+  const { control, handleSubmit, getValues, setValue } = methods;
+
+  const [step, setStep] = useState(ConfigurationsSteps.LorisConfigurations);
+  const [projects, setProjects] = useState([]);
+  const [error, setError] = useState<string | undefined>(undefined);
+
+  const dispatch = useAppDispatch();
+  const { result: appletData } = applet.useAppletData() ?? {};
+  const { updateAppletData } = applet.actions;
+
+  const saveConfiguration = async () => {
+    try {
+      const { hostname, username, password } = getValues();
+      const projects = await fetchLorisProjects(hostname, username, password);
+      setProjects(projects);
+      setStep(ConfigurationsSteps.SelectProject);
+      setError(undefined);
+    } catch (error) {
+      setError(t('loris.errors.fetchProjectsFailed'));
```

Comment:
This error message is different than figma

| Figma | Actual |
|-|-|
| ![CleanShot 2024-10-17 at 16 04 40](https://github.com/user-attachments/assets/7b44ea8b-38d9-4ca6-a852-f4d0216f5a19) | ![image](https://github.com/user-attachments/assets/4e107e24-be85-46cc-9122-5a53bb8fe3f6) |